docs/developer/proposal/index.md

```diff
@@ -0,0 +1,40 @@
+# Enhancement Proposals
+
+This directory contains Enhancement Proposals (EPs) for major features and changes to Kepler.
+
+## Active Proposals
+
+| ID | Title | Status | Author | Created |
+|----|-------|--------|--------|---------|
+| [EP-000](EP_TEMPLATE.md) | Enhancement Proposal Template | Accepted |Sunil Thaha | 2025-01-18 |
+
+## Proposal Status
```

Comment:
I suppose we may no need this, for example a proposal in a PR, the PR merged means Accepted?

Reply:
Thats right